corecito.py

```diff
@@ -33,14 +35,26 @@ async def main():
 
       logger.info(f"Balances\n(Base) {account.base_currency} balance:{balances['base_currency_balance']} \n(Core) {account.core_number_currency} balance:{balances['core_number_currency_balance']}\n")
 
-
       ###########################
       # Core Number Adjustments #
       ###########################
-      deviated_core_number = balances['base_currency_available'] / buy_price
+      
+      # if 'fiat', balances are calculated by multiplying buy_price
+      if fiat:
+        deviated_core_number = balances['base_currency_available'] * buy_price
+      else:
+        deviated_core_number = balances['base_currency_available'] / buy_price
+
       logger.info(f'Core number adjustments')
-      logger.info(f'Core number: {account.core_number} {account.core_number_currency}')
-      logger.info(f'Deviated Core number:{deviated_core_number:.6f} {account.core_number_currency}')
+      
+      #Adjust logger.info text format if 'fiat'
+      if fiat:
+        logger.info(f'Core number: €{account.core_number}')
```

Comment:
Why is the `account.core_number_currency` not shown for fiat? Is the core_number_currency in this case not EUR?

Reply:
This is just styling the text format to replace 'EUR' for the euro symbol €. But you are right, as the code should be generic for any fiat or stable coin I will update this.

Reply:
Updated!